src/data/data1d/Spectrum1D/ranges/resurrectSpectrum.ts

```diff
@@ -94,7 +99,10 @@ function resurrectSpectrumFromSignals(
           isFt: true,
           name,
         },
-        ranges: { values: signals, options: { sum: 100 } },
+        ranges: {
+          values: signalsToRanges(signalsJoin(signals)),
+          options: { sum: 100 },
```

Comment:
Should probably be the sum of the signals integration or the sum of the atoms if available ?